JESCorrections/test/fitJESCs

```diff
@@ -158,23 +161,36 @@ runJECWorflowForOneAlgo(){
   stepNameOld=${stepName} && ((++stepN)) && stepName="step"$(printf "%02d" ${stepN})
   if [ ! -f ${stepName}.done ]; then
     CMD_STEP="jet_response_analyzer_x ${JESC_BASE}/config/jra_hltRun3.config -input ${JRANTuple_PU} -algs ${jetType}:${dRgen}"
-    CMD_STEP+=" -levels 1 -path ./ -era ${ERA} -output plots_${stepName}/histogram_${jetType}l1_${stepName}.root -maxEvts ${maxEvents}"
+    CMD_STEP+=" -era ${ERA} -output plots_${stepName}/histogram_${jetType}l1_${stepName}.root -maxEvts ${maxEvents}"
     CMD_STEP+=" -useweight false -nrefmax 50 -relrspmin 0.0 -relrspmax 5.0 -nbinsabsrsp 0 -nbinsetarsp 0 -nbinsphirsp 0 -nbinsrelrsp 200 -doflavor false"
+    if [ "X${skipL1}" != "Xtrue" ]; then
+      CMD_STEP+="  -path ./ -levels 1  "
+    else
+      echo '!!!skipping l1!!!'
+    fi
     stepCmd "mkdir -p plots_${stepName}\n\n${CMD_STEP}" ${stepName}
   fi
 
   stepNameOld=${stepName} && ((++stepN)) && stepName="step"$(printf "%02d" ${stepN})
   if [ ! -f ${stepName}.done ]; then
     CMD_STEP="jet_l2_correction_x -algs ${jetType} -era ${ERA} -l2l3 true"
     CMD_STEP+=" -input plots_${stepNameOld}/histogram_${jetType}l1_${stepNameOld}.root -outputDir ./ -output l2p3.root"
-    CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+gaussian -maxFitIter 50 -ptclipfit true -ptclip 10 -statTh 10 -useOfflinePFFunctions true"
+    CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+gaussian -maxFitIter 50 -ptclipfit true" # for all other jet collections
+    # for ak8puppi only
+    #CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+Gaussian -maxFitIter 50 -ptclipfit false" # for ak8puppi only
```

Comment:
next time, let's put a `if` block based on `${jetType}`, so that the code runs for all collections without caveats